plugins/data-management/src/HierarchicalTrackSelectorWidget/components/tree/HamburgerMenu.tsx

```diff
@@ -93,22 +82,34 @@ export default observer(function HamburgerMenu({
 
   return (
     <>
-      <IconButton
-        className={classes.menuIcon}
-        onClick={event => setMenuEl(event.currentTarget)}
-      >
-        <MenuIcon />
-      </IconButton>
-
-      <JBrowseMenu
-        anchorEl={menuEl}
-        open={Boolean(menuEl)}
-        onMenuItemClick={(_, callback) => {
-          callback()
-          setMenuEl(undefined)
-        }}
-        onClose={() => setMenuEl(undefined)}
+      <CascadingMenuButton
         menuItems={[
+          {
+            label: 'Sort tracks by name',
+            type: 'checkbox',
+            checked: model.hSortTrackNames,
+            onClick: () => model.setSortTrackNames(!model.hSortTrackNames),
+          },
+          {
+            label: 'Sort categories by name',
+            type: 'checkbox',
+            checked: model.hSortCategories,
+            onClick: () => model.setSortCategories(!model.hSortCategories),
+          },
+          { type: 'divider' },
+          {
+            label: 'Collapse subcategories',
+            onClick: () => model.collapseSubCategories(),
+          },
+          {
+            label: 'Collapse top-level categories',
+            onClick: () => model.collapseTopLevelCategories(),
+          },
+          {
+            label: 'Expand all categories',
```

Comment:
Overall I don't think there's enough distinction between what these options do...

Collapse top-level categories does the same thing that subcategories does on the demo link?

They should all be toggles to expand all/collapse all

Reply:
They are slightly different behaviors, see the behavior on the volvox example. possibly the subcategories one could be hidden if no subcategories are even available on track selector

Reply:
I now removed the subcategories option if there are none, which may remove some cognitive overload from the unnecessary option